src/config/conf.dev.js

```diff
@@ -284,7 +284,9 @@ const config = {
     ui: 'bdd',
     reporter: 'spec',
     enableTimeouts: false,
-        // ,bail: true,
+    reporterOptions: {
+      html: true,
```

Comment:
We have changed to AirBnB code style for the OSS repos and this enforces this :-)